src/main/diagnostics/__tests__/diagnostics.spec.ts

```diff
@@ -0,0 +1,67 @@
+import yaml from 'yaml';
+
+import { Diagnostics } from '../diagnostics';
+
+describe(Diagnostics, () => {
+  const diagnosticsTable = `
```

Comment:
This is so going to be out of date soon. Can we just read the file out of `assets/` instead?
(Otherwise, Jest has support for [snapshot testing](https://jestjs.io/docs/snapshot-testing).)

Reply:
No, I copied the YAML file into the test with the idea that we don't have to update the sample code in the test. Comment added.

Reply:
In that case, it might have been better to use obviously-testing data, instead of copying production data.

Probably not making extra commits just for this, but please remember for similar situations in the future.